bootstrap/bootstrap.sh

```diff
@@ -28,8 +29,32 @@ sudo apt-get update
 sudo apt-get -y dist-upgrade
 
 #----nodejs install
-curl -sL https://deb.nodesource.com/setup_7.x | sudo -E bash -
+curl -sL https://deb.nodesource.com/setup_6.x | sudo -E bash -
 sudo apt-get install -y nodejs
 
 #----alsa install
 sudo apt-get install -y alsa-base alsa-utils
+
+#----install pigpio (npm install pigpio will be exec by tjbotlib package.json)
+curl abyz.co.uk/rpi/pigpio/pigpio.zip > pigpio.zip
```

Comment:
> It has been my experience on some pi's that pigpio did not get installed for some reason ...

To the best of my knowledge Raspbian Jessie 2016-05-10 or newer comes with pigpio pre-installed. Earlier version of Raspbian don't come with pigpio pre-installed. Raspbian Jessie Lite doesn't come with pigpio pre-installed either.

Were you by any chance using Raspbian Jessie sometimes and Raspbian Jessie Lite other times?